src/isIsogram.js

```diff
@@ -20,3 +15,14 @@ function isIsogram(word) {
 }
 
 module.exports = { isIsogram };
+
+
+function maxValue(array) {
+  return Math.max(...arr);
+}
```

Comment:
This function `maxValue` is not related to the `isIsogram` function, and it's not exported. If it's not needed for this file, consider removing it to keep the code clean and focused on its purpose. If it's needed elsewhere, consider moving it to a more appropriate location.